crates/polars-io/src/csv/read/read_impl.rs

```diff
@@ -206,6 +206,11 @@ impl<'a> CoreReader<'a> {
             },
         };
         if let Some(dtypes) = dtype_overwrite {
+            if dtypes.len() > schema.len() {
```

Comment:
Nit: We can use `polars_ensure` macro. A little bit cleaner.

Reply:
thanks, updated.